### DISCUSSION TITLE
Lua VM Crashed error

### DISCUSSION THREAD

**Discussion by kollol-chowdhury-incontact** [Kong/kong#8550] [Help]
Using Kong 2.7.0 and very frequently getting the error: 
nginx: lua atpanic: Lua VM crashed, reason: not enough memory

the mem_cache size set is something around 500MB.
Is this causing the issue?

What's the difference in memory utilisation between kong 2.0.3 and kong 2.7.0?

Please help

**Comment by kollol-chowdhury-incontact** [comment]
@kikito :Could you please help

**Reply by StarlightIbuki** [reply]
Could you provide more details? For example, what configuration you are using, and usually what operation you are doing when it crashes? What is the size of your physical memory and how much memory does the gateway consume before crashes? Thank you.

**Reply by kollol-chowdhury-incontact** [reply]
We are using c5.2xlarge instances.
